db/migrate/20231222202341_like_table.rb

```diff
@@ -0,0 +1,9 @@
+class LikesTable < ActiveRecord::Migration[7.1]
```

Comment:
- [ ] **[OPTIONAL]** You should rename this file to `...likes_table.rb` to match the class name.

  ![image](https://github.com/Sami-Ullah-Tufail/blog/assets/36542986/cc2a8939-67a7-4f35-a979-0800eb8818cf)